SR2024/2024-03-15-competition-details.md

```diff
@@ -0,0 +1,63 @@
+---
+to: Student Robotics 2024
+subject: SR2024 Competition Details
+attachment: https://drive.google.com/file/d/1JQG_IKOtnvEUPSob1XWTma55FHRK_r4o/view
+---
+
+Hello everyone!
+
+It's just under a month until the big day - [the competition][competition-event-page]!
+
+Before then, we have a few important updates and reminders for you:
+
+## Attendance details
+
+Thanks to everyone who has already provided their attendance details. If you haven't done that yet, or things have changed substantially please [provide attendance details](https://forms.gle/8YhfdRbfDWP3KKNy8).
+
+Doors open on both days at 9am and each day will finish around 5:30pm. The event will formally start at 10:30am on Saturday. There are now full day schedules on the [event page][competition-event-page]. **If you are expecting to arrive after 10am on Saturday, please let us know.**
+
+You'll have access to your pits and the test arena when you arrive, giving your team time to work on their robot before things get going in earnest.
```

Comment:
I haven't mentioned the expectation that we'll require a safety check before allowing them into the test arena. I'm not sure that detail is needed here, though it's mostly that I couldn't think of a good wording which didn't get distracted by adding it. I doubt anyone is going to complain if we require a safety check but haven't mentioned it -- the goal of this sentence is to get them excited about arriving early, rather than being a full description of the process.